g-local_vce_detailed.md

```diff
@@ -318,3 +318,32 @@ The following flags may also be used with the `docker run` command:
 | Automatically remove container when it exits  | `--rm`                                                   |
 | Run container in background      | `-d/--detach`                                            |
 | Set environment variable inside  container     | `-e/--env ENV_VARIABLE="ENV VALUE"`                      |
+## Updating the Docker application
+It is good practice to keep your version of Docker Desktop up to date. If there are updates to Docker Desktop, the *Update to latest* button will appear to the left of the Docker Desktop search toolbar, {numref}`docker_desktop_update`.
+
+```{figure} md_assets/media/docker_update.png
+:name: docker_desktop_update
+:width: 5.44791666666667in
+:height: 2.041666666667in
+
+Checking for updates in Docker Desktop
+
+```
+It is possible to update to the latest version by doing the following.
+- Click on the *Update to latest* button. This will take you to the *Softwares updates* page.
+- Click on *Download update* and then *Update and restart*.
+
+Note: we strongly recommend that you **do not update the software within the VCE unless told to do so by the module team**. Software updates and upgrades occasionally introduce changes that result in undesirable software behaviour, sometimes known as ‘breaking changes’, that cannot always be predicted in advance.
+
+## Uninstalling the Docker application
+When you no longer require the Docker application to run the VCE (or VCEs for any other module you study), it is recommend that that you uninstall the application. 
```

Comment:
recommend -> recommended

I disagree with the sense of this. I think we should be encouraging students to use docker more widely... How about:

When you no longer require the Docker application to run the VCE (or VCEs for any other module you are studying), you can uninstall it from your device.

Reply:
Yes I'm happy to go with that. Particularly as students may not know if it will pop up for one of their later modules. I suspect it would make many students' lives easier if module software was generally deployed using containers.